src/pages/Components/Tags/design.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'Design'
+order: 4
+---
+
+## Tags sketch file is composed of three sections :
+
+The sketch file includes other form elements. Here is the section you'll find the radio.
```

Comment:
This is not a form element, I don't understand this sentence

Reply:
It's just a typo because the design page is taken from other pattern, it will be fixed. No worries